packages/sources/src/components/navbar/navbar.args.ts

```diff
@@ -0,0 +1,42 @@
+import { ArgTypes } from '../../stories-helpers';
+
+import { Navbar, NavbarItem } from './navbar.models';
+
+export interface NavbarArgs {
+  href: string;
+  items: NavbarItem[];
+  modifier: string;
+  open: boolean;
+}
+
+export const navbarArgTypes: ArgTypes<NavbarArgs> = {
+  href: {
+    control: {
+      type: 'string'
+    }
+  },
+  items: {
+    control: {
+      disable: true
+    }
+  },
+  modifier: {
+    control: {
+      type: 'string'
+    }
+  },
+  open: {
+    control: {
+      type: 'boolean'
+    }
+  }
+};
+
+export function navbarArgsMapper(a: NavbarArgs): Navbar {
+  return {
+    href: a.href,
```

Comment:
de navbar ZELF heeft geen href! De items hebben dat.

Reply:
Zo, ik moest even verder kijken dan mijn neus lang was. Is inmiddels aangepast.